server/weaverbird/backends/pipeline_translator/steps/utils/query_transformation.py

```diff
@@ -0,0 +1,23 @@
+from numpy import logical_and, logical_or
+
+from server.weaverbird.pipeline.conditions import Condition, ComparisonCondition, InclusionCondition, NullCondition, \
+    MatchCondition, ConditionComboAnd, ConditionComboOr
+
+
+def apply_condition(condition: Condition, query: str) -> str:
+
+    query = f'select * from ({query}) '
+    sql_operator = {
+        'eq': '=',
+        'ne': '!=',
+        'lt': '<',
+        'le': '<=',
+        'gt': '>',
+        'ge': '>=',
+    }
```

Comment:
You can put this in a constant outside the function, so it will not be reallocated on each function call.

Reply:
Great, thanks !